pom.xml

```diff
@@ -475,7 +475,7 @@
     <version.org.awaitility>3.0.0</version.org.awaitility>
     <version.io.github.bonigarcia>3.8.1</version.io.github.bonigarcia>
 
-    <version.org.kogito.gwt-jsonix-schema-compiler>1.2.0</version.org.kogito.gwt-jsonix-schema-compiler>
+    <version.org.kogito.gwt-jsonix-schema-compiler>1.2.1-SNAPSHOT</version.org.kogito.gwt-jsonix-schema-compiler>
```

Comment:
Shouldn't be 1.2.1 instead of 1.2.1-SNAPSHOT? Asking because of https://github.com/kiegroup/gwt-jsonix-schema-compiler/pull/8/files

Reply:
@danielzhe @jomarko Version `1.2.1` is now released. You can set version `1.2.1` Thanks!